physics/rainfall_intensity.py

```diff
@@ -0,0 +1,115 @@
+"""
+Rainfall Intensity
+==================
+This module contains functions to calculate the intensity of
+a rainfall event for a given duration and return period.
+"""
+
+
+def rainfall_intensity(
+    k: float, a: float, b: float, c: float, tr: float, t: float
+) -> float:
+    """
+    Calculate the intensity of a rainfall event for a given duration and return period.
+    The coefficients K, a, b, and c are obtained from the Sherman
+    intensity-duration-frequency curve for a specific location.
+
+    Parameters
+    ----------
+    k : float
+        Coefficient [adm].
+    a : float
+        Coefficient [adm].
+    b : float
+        Coefficient [adm].
+    c : float
+        Coefficient [adm].
+    tr : float
+        Return period in years.
+    t : float
+        Rainfall event duration in minutes.
```

Comment:
Could you rename the last two variables to be more descriptive? That is, `t` -> `duration`, `tr` -> `return_period`, or something similar.